api/envoy/extensions/filters/http/router/v3/router.proto

```diff
@@ -21,11 +21,18 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Router :ref:`configuration overview <config_http_filters_router>`.
 // [#extension: envoy.filters.http.router]
 
-// [#next-free-field: 9]
+// [#next-free-field: 10]
 message Router {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.router.v2.Router";
 
+  message UpstreamAccessLogOptions {
+    // If set to true, an upstream access log will recorded once when a new HTTP request is received,
+    // and an upstream connection has been successfully established. Note that in case a retry is applied
```

Comment:
It is unclear from the comment what happens if a new stream is created to an already established connection. I'm assuming it will also be flushed, but the wording is a bit confusing.

Reply:
done

Reply:
done